### PR TITLE
Contact edit view

### PR DESCRIPTION
**Pull request by thdvbr**
- moved emergency feature from home screen
- add contacts enabled when user has none 
- added edit and delete contact screen, functionalities

### CODE REVIEW

File: src/components/sosContacts/SosContactList.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import { useNavigation } from '@react-navigation/native';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+
+import { useIsFocused } from '@react-navigation/native';
+import { Context as AuthContext } from '../../state/AuthContext';
+import appApiClient from '../../api/appApiClient';
+
+export default function SosContactList() {
+  const navigation = useNavigation();
```
Comment (irina-baeva) [inline src/components/sosContacts/SosContactList.js:13]: why we need useNavigation here we cannot use navigation from props ?

File: src/screens/SosContactEdit/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  };
+
+  const handleRemove = async (id) => {
+    appApiClient
+      .delete(`/users/${state.username}/contacts`, {
+        params: { id },
+      })
+      .then((response) => {
+        console.log(response.data);
+      })
```
Comment (irina-baeva) [inline src/screens/SosContactEdit/index.js:76]: console log response? may be better to do something like alerting that "contact was deleted"?

**Review by irina-baeva — APPROVED** [review]

File: src/screens/SosContactEdit/index.js (hunk 2 above)
Comment (thdvbr) [reply src/screens/SosContactEdit/index.js:76]: response.data is a message that says contact was successfully deleted, ill change to alert

File: src/components/sosContacts/SosContactList.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
+/* eslint no-underscore-dangle: ['error', { 'allow': ['_id'] }] */
+import React, { useContext, useEffect, useState } from 'react';
+import { StyleSheet, Button, Text, TouchableOpacity } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+
+import { useIsFocused } from '@react-navigation/native';
+import { Context as AuthContext } from '../../state/AuthContext';
```
Comment (irina-baeva) [inline src/components/sosContacts/SosContactList.js:9]: then if we stay with hooks, lets import in one row  useIsFocused and useNavigation

**Review by irina-baeva — APPROVED** [review]

**Pull request merged**